src/main/java/uk/gov/hmcts/probate/config/ApplicationConfiguration.java

```diff
@@ -36,7 +38,9 @@ public ResourceBundleMessageSource resourceMessageSource() {
 
     @Bean
     public RestTemplate restTemplate() {
-        return new RestTemplate();
+        RestTemplate restTemplate = new RestTemplate();
+        restTemplate.setRequestFactory(new HttpComponentsClientHttpRequestFactory(getHttpClient()));
```

Comment:
I believe Feign uses the restTemplate declared here - thats why you have needed to add the factory in there. Did all your regressions work OK?

Reply:
I think it would be better if you can set this configuration directly on feign without manipulating rest template - it will require some research